examples/node/src/invitations.ts

```diff
@@ -0,0 +1,27 @@
+// Usage:
+// From examples/node, transpile files by running `tsc`
+// To run:
+// node --require dotenv/config dist/invitations.js
+
+import { setClerkServerApiUrl, invitations } from '@clerk/clerk-sdk-node';
+
+const serverApiUrl = process.env.CLERK_API_URL || '';
+
+setClerkServerApiUrl(serverApiUrl);
+
+console.log('Create an invitation');
```

Comment:
❓ Do we need these `console.log` statements? Let's remove them.

Reply:
Done. Kept only the last one, so you can see that the revoked invitation is not returned in the list.